### ISSUE TITLE
Problem with epub Metadata "No metadata avalible" V2

### ISSUE THREAD

**Issue by Rougecarlos** [Kareadita/Kavita#1411]
I has make a previusly issue, and you can't reproduce it yourself this https://github.com/Kareadita/Kavita/issues/1361

The problem is that Kavita dont cath metadata of the epubs in windows, but with docker, works well.

I was thinking about it, and try a few things, and finally i think i have the problem is my Windows Language, with the system in Spanish, kavita dont catch metadata, but if i change it to english in windows works well. I made a video with the problem for if you want to see it.
https://www.youtube.com/watch?v=JsgayKkeeP0&feature=youtu.be
Basicly:

1º New Install of kavita with system in Spanish
2º Scan folder with 1 epub, and metadata are miss.
3º Rescan the epub and is the same no metadata
4º Delete de Epub and Scan with the same result
5º Change the system language to English (United States)
6º Scan the folder and all metadata are there!

I dont know if you can do something, to the metadata for show it in Spanish

**Comment by majora2007** [comment]
Hmm...interesting. I know we've seen some weird regex parsing on Italian systems. Let me as @ThePromidius to take a look, since he has Spanish on his OS.

**Comment by ThePromidius** [comment]
I have my laptop so will try it out later. I'm assuming it's plain windows right? No docker

**Comment by majora2007** [comment]
@ThePromidius Yes. Plain windows with a Spanish OS language set.

**Comment by ThePromidius** [comment]
TLDR: Can't replicate. 
Created new instance on windows and added one test library with one epub file. Filesystem loc is spanish.
Added the library and kavita auto-scanned. It picked up the epub metadata on first try (title, author and publisher)

**Comment by Rougecarlos** [comment]
I'm cursed ^ ^. Can you try it with this epub? It's the one I use in tests.

https://mega.nz/file/ZHlFzAgQ#1Wncjvh7MetbBkdG8LI_ZLkxsNDLkZtF7E8UTh8dqO8

**Comment by ThePromidius** [comment]
Tried with that file and this is all the metadata that is loaded:
![imagen](https://user-images.githubusercontent.com/34222883/183600634-6e4bdba5-6d80-4c55-a924-9106b6296a21.png)

 metadata inside the epub file under `OEBPS/content.opf`
 ```
<package xmlns="http://www.idpf.org/2007/opf" version="3.0" unique-identifier="pub-id" xml:lang="en" prefix="calibre: https://calibre-ebook.com">
  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/"  xmlns:opf="http://www.idpf.org/2007/opf" xmlns:calibre="http://calibre.kovidgoyal.net/2009/metadata">
    <dc:title>86 - EIGHTY-SIX, Vol. 01</dc:title>
    <dc:subject>Fiction</dc:subject>
    <dc:subject>Science Fiction</dc:subject>
    <dc:subject>Military</dc:subject>
    <meta name="calibre:title_sort" content="86 - EIGHTY-SIX, Vol. 01"/>
    <dc:creator id="creator2">Shirabii</dc:creator>
    <meta refines="#creator2" property="file-as">Shirabii</meta>
    <dc:creator id="id-2">Asato Asato</dc:creator>
    <dc:creator id="id-3">Shirabii</dc:creator>
    <dc:rights>English copyright © 2019 by Yen Press, LLC</dc:rights>
    <dc:identifier>isbn:9781975303136</dc:identifier>
    <dc:identifier>google:NYeFDwAAQBAJ</dc:identifier>
    <dc:identifier>calibre:276</dc:identifier>
    <dc:identifier>uuid:88be7e33-c788-4d52-afc5-ff9cca54cf59</dc:identifier>
    <dc:identifier id="pub-id">uuid:fcfb48b6-c873-497e-ad8c-8224a7b28c4a</dc:identifier>
    <dc:language>en</dc:language>
    <dc:contributor id="id-1">calibre (5.44.0) [https://calibre-ebook.com]</dc:contributor>
    <dc:date>2019-03-25T23:00:00+00:00</dc:date>
    <dc:description>&lt;div&gt;&lt;p&gt;A War Without CasualtiesThe Republic of San Magnolia has long been under attack from the neighboring Giadian Empire&apos;s army of unmanned drones known as the Legion. After years of painstaking research, the Republic finally developed autonomous drones of their own, turning the one-sided struggle into a war without casualties-or at least, that&apos;s what the government claims.In truth, there is no such thing as a bloodless war. Beyond the fortified walls protecting the eighty-five Republic territories lies the &quot;nonexistent&quot; Eighty-Sixth Sector. The young men and women of this forsaken land are branded the Eighty-Six and, stripped of their humanity, pilot the &quot;unmanned&quot; weapons into battle...&lt;/p&gt;&lt;/div&gt;</dc:description>
    <dc:publisher>Yen Press</dc:publisher>
    <opf:meta refines="#id" property="title-type">main</opf:meta>
    <opf:meta refines="#id" property="file-as">86 - EIGHTY-SIX, Vol. 01</opf:meta>
    <meta refines="#creator2" property="role" scheme="marc:relators">ill</meta>
    <meta property="dcterms:modified">2019-03-28T10:01:07Z</meta>
    <meta name="Sigil version" content="0.9.9"/>
    <meta content="9781975303136.jpg" name="cover"/>
    <opf:meta refines="#id-1" property="role" scheme="marc:relators">bkp</opf:meta>
    <opf:meta refines="#id-2" property="role" scheme="marc:relators">aut</opf:meta>
    <opf:meta refines="#id-2" property="file-as">Asato, Asato</opf:meta>
    <opf:meta refines="#id-3" property="role" scheme="marc:relators">aut</opf:meta>
    <opf:meta refines="#id-3" property="file-as">Shirabii</opf:meta>
    <opf:meta property="belongs-to-collection" id="id-4">86 - Eighty Six</opf:meta>
    <opf:meta refines="#id-4" property="collection-type">series</opf:meta>
    <opf:meta refines="#id-4" property="group-position">1</opf:meta>
    <opf:meta property="calibre:author_link_map">{"Asato Asato": "", "Shirabii": ""}</opf:meta>
  </metadata>
```

**Comment by ThePromidius** [comment]
This windows version won't let me change language set lol
